tools/lite_interpreter/gen_oplist.py

```diff
@@ -0,0 +1,199 @@
+            + "model YAML files that contain the set of used operators. This "
+            + "file path must have a leading @-symbol, which will be stripped "
+            + "out before processing."
+        ),
+        required=False,
+    )
+    parser.add_argument(
+        "--yaml_file_path",
+        help=(
+            "Path to a yaml file"
+        ),
+        required=True,
+    )
+    parser.add_argument(
+        "--allow_include_all_overloads",
+        help=(
+            "Flag to allow operators that include all overloads. "
+            + "If not set, operators registered without using the traced style will"
+            + "break the build."
+        ),
+        action="store_true",
+        default=False,
+        required=False,
+    )
+    options = parser.parse_args()
+
+    # assert options.model_file_list_path[0] == "@"
+    # model_file_list_path = options.model_file_list_path[1:]
+
+    # model_dicts = []
+    # with open(model_file_list_path) as model_list_file:
+    #     model_file_names = model_list_file.read().split()
+    #     for model_file_name in model_file_names:
+    #         with open(model_file_name, "rb") as model_file:
+    #             model_dict = yaml.safe_load(model_file)
+    #             model_dicts.append(model_dict)
+    model_dicts = []
+    model_dict = yaml.safe_load(open(options.yaml_file_path))
+    model_dicts.append(model_dict)
+    print("model_dict: ")
+    print(model_dict)
+    print("options.yaml_file_path: ", options.yaml_file_path)
+    selective_builders = list(
+        map(
+            lambda m: SelectiveBuilder.from_yaml_dict(m),
+            model_dicts,
+        )
+    )
+
+    # While we have the model_dicts generate the supported mobile models api
+    gen_supported_mobile_models(model_dicts, options.output_dir)
+
+    # We may have 0 selective builders since there may not be any viable
+    # pt_operator_library rule marked as a dep for the pt_operator_registry rule.
+    # This is potentially an error, and we should probably raise an assertion
+    # failure here. However, this needs to be investigated further.
+    selective_builder = SelectiveBuilder.from_yaml_dict({})
+    if len(selective_builders) > 0:
+        selective_builder = reduce(
+            combine_selective_builders,
+            selective_builders,
+        )
+
+    if not options.allow_include_all_overloads:
+        throw_if_any_op_includes_overloads(selective_builder)
+    with open(
+        os.path.join(options.output_dir, "selected_operators.yaml"), "wb"
+    ) as out_file:
+        out_file.write(
+            yaml.safe_dump(
+                selective_builder.to_dict(), default_flow_style=False
+            ).encode("utf-8"),
+        )
+
+    write_selected_mobile_ops(
+        os.path.join(options.output_dir, "selected_mobile_ops.h"),
+        selective_builder,
+    )
+
+
+if __name__ == "__main__":
+    sys.exit(main(sys.argv))
```

Comment:
I haven't noticed a single return statement in implementation of `main` routine
```suggestion
    main(sys.argv)
```